Clean up Vale errors in task topics (#922)
grafana/alloy: docs/sources/reference/components/otelcol.auth.oauth2.md

# otelcol.auth.oauth2

`otelcol.auth.oauth2` exposes a `handler` that can be used by other `otelcol` components to authenticate requests using OAuth 2.0.

The authorization tokens can be used by HTTP and gRPC based OpenTelemetry exporters.
This component can fetch and refresh expired tokens automatically.
Refer to the [OAuth 2.0 Authorization Framework](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4) for more information about the Auth 2.0 Client Credentials flow.

{{< admonition type="note" >}}
`otelcol.auth.oauth2` is a wrapper over the upstream OpenTelemetry Collector `oauth2client` extension.
Bug reports or feature requests will be redirected to the upstream repository, if necessary.
{{< /admonition >}}

> Youc an specify multiple `otelcol.auth.oauth2` components by giving them different labels.

Author: ```suggestion
You can specify multiple `otelcol.auth.oauth2` components by giving them different labels.
```
